catkin_ws/src/image_rotation/src/image_rotation_publisher.cpp

```diff
@@ -0,0 +1,47 @@
+///
+/// @file
+/// @brief Implementation of ROS Publisher
+///
+
+#include <iostream>
+
+#include "../include/image_rotation/image_algorithm.h"
```

Comment:
Add include directory in CMake to avoid relative paths.Let " image_rotation/image_algorithm.h" be the convention

Reply:
iostream not required

Reply:
done